android/app/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,20 @@
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.VIBRATE"/>
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
+    <queries>
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <data android:scheme="https" />
```

Comment:
should we add another `<intent` for `http`? Just thinking if HTTP links added by users for their websites or personal links will work.

Reply:
http links are also working with current changes